Purchasing.Mvc/Services/OrderService.cs

```diff
@@ -202,19 +202,23 @@ public async Task CreateApprovalsForNewOrder(Order order, int[] conditionalAppro
                     {
                         if (FinancialChartValidation.GetFinancialChartStringType(accountId) == FinancialChartStringType.Invalid)
                         {
-                            var externalAccount = _repositoryFactory.AccountRepository.GetNullableById(accountId);
-
+                            //var externalAccount = _repositoryFactory.AccountRepository.GetNullableById(accountId);
+
+                            //approvalInfo.Approver = null;
+                            //approvalInfo.AcctManager = externalAccount != null
+                            //                               ? _securityService.GetUser(externalAccount.AccountManagerId)
+                            //                               : null;
+                            //approvalInfo.Purchaser = null;
+                            
```

Comment:
delete me

Reply:
Yeah, I know. I can do that later, but I want to keep it around in case something breaks.